Maquetado/scss/bookDetail.scss

```diff
@@ -2,88 +2,117 @@
 @import 'mixins';
 
 .bookDetailBackground {
-    align-items: center;
-    background-color: $seashell;
+  align-items: center;
+  background-color: $seashell;
+  display: flex;
+  flex-direction: column;
+  gap: 15px;
+  padding-top: 25px;
+  padding-bottom: 28%;
```

Comment:
Have you tried to replacing this with a `height:100vh`? padding should be used to separate a container from it's content